tutorials/colors/colormapnorms.py

```diff
@@ -188,8 +188,8 @@
 
 
 ###############################################################################
-# DivergingNorm: Different mapping on either side of a center
-# -----------------------------------------------------------
+# TwoSlopeNorm: Different mapping on either side of a center
+# ---------------------------------------------------------
```

Comment:
Aha! There is the "too short underline".

Reply:
My first pass at this I spelled it `TwoSlopNorm` and apparently did not fix up all of the fixes...